### ISSUE TITLE
How to add LoongArch machine support

### ISSUE THREAD

**Issue by Panxuefeng-loongson** [adoptium/infrastructure#2806]
Loongson Technology wants to participate in the Adoptium community. Currently, we are submitting a PR for the Loongson platform JDK (Here is our open-sourced repo on github: https://github.com/loongson/jdk8u/tree/master-ls, including LoongArch and MIPS support), see: https://github.com/adoptium/temurin-build/pull/3134. Although the LoongArch platform JDK can be built by cross-compiling, but in the Adoptium community (such as Jenkins), it is definitely more convenient to have LoongArch's real machine/physical machine for testing and other activities. Loongson Technology is willing to provide LoongArch real machine/physical machine to Adoptium. How should I do it? Do you have any suggestions and any documents for reference?

**Comment by Panxuefeng-loongson** [comment]
@smlambert @sxa @theaoqi I am sorry to disturb you all. I would like to ask for suggestions and move the issue forward.

**Comment by sxa** [comment]
Based on an earlier discussion with @tellison it sounds like we are not in a position to support this in our CI until such time as the loongarch port is merged into the upstream openjdk codebase. Once this is in place we can consider adding it.
